pkg/acme/cert_request.go

```diff
@@ -37,6 +37,7 @@ func (a *Acme) ensureAcmeClient() error {
 	account, err := a.validateUser(client, accountURI)
 	if err != nil {
 		a.Log().Fatalf("fatal error verifying existing user: %s", err)
```

Comment:
The `Fatal` log functions will automatically call `os.Exit(1)` after their execution, so the new `return err` line is actually unreachable. Can you change the `Fatalf` line to call `Errorf` instead? This should hopefully fix your issue!